tests/testthat/helper-locale.R

```diff
@@ -0,0 +1,8 @@
+is_locale_available <- function(loc){
```

Comment:
If this function succeeds, it doesn't just test whether it's possible to set the locale -- it will also set the locale to the new one. See note below...

Reply:
Good catch! Fixed now, I believe.